contracts/tokens/ERC721NativeToken.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity 0.7.5;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "../interfaces/IOwnable.sol";
+
+contract ERC721NativeToken is ERC721 {
```

Comment:
It should be ERC721BridgableToken ?

Reply:
- `ERC721NativeToken` deployed by the user 
- `ERC721BridgableToken` deployed by bridge after user bridging first NFT

Reply:
I mean that this token is the customized token by G.U.? Then we might be change the class name to more clear one. GUERC771NativaToken, etc. How do you think?

Reply:
That's oke. But can I update it after merge because depend?